data.js

```diff
@@ -15,6 +15,14 @@ module.exports = [
     url: "https://github.com/rvanbaalen/hashparser",
     source: "https://raw.githubusercontent.com/rvanbaalen/hashparser/master/src/hashparser.js"
   },
+  {
+    name: "Carbonium",
+    github: "edwinm/carbonium",
+    tags: ["dom", "qsa", "jquery", "frontend", "javascript", "typescript"],
+    description: "One kilobyte library for easy manipulation of the DOM.",
+    url: "https://github.com/edwinm/carbonium",
+    source: "https://raw.githubusercontent.com/edwinm/carbonium/master/src/carbonium.ts"
```

Comment:
Please link to a JavaScript source file, not a TypeScript source file.

From the readme - emphasis on "ready-to-use":

> The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified (see below)

Reply:
Thanks for pointing this out. The source is updated.